core/src/main/java/org/mule/config/DefaultMuleConfiguration.java

```diff
@@ -204,6 +204,11 @@
      */
     private ProcessingStrategy defaultProcessingStrategy;
 
+    /**
+     * {@inheritDoc}
```

Comment:
No, you are right. I thought you were pointing out the getter. I will remove it.